apps/server/src/modules/task-card/controller/mapper/task-card.mapper.ts

```diff
@@ -24,14 +24,15 @@ export class TaskCardMapper {
 			visibleAtDate: card.visibleAtDate,
 			dueDate: card.dueDate,
 			title: card.title,
+			courseId: card.course.id,
+			courseName: card.course.name,
 		});
 		if (card.cardElements.length) {
 			dto.cardElements = this.getCardElementResponse(card);
 		}
-		if (card.course) {
-			dto.courseId = card.course.id;
-			dto.courseName = card.course.name;
-		}
+
+		dto.courseId = card.course.id;
+		dto.courseName = card.course.name;
```

Comment:
why not directly putting courseId and courseName directly into the declaration? It was just outside due to the optionality